server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/burstcoin/sources/poloniex/PoloniexRateSource.java

```diff
@@ -33,19 +32,14 @@
 public class PoloniexRateSource implements IRateSource{
     private static final Logger log = LoggerFactory.getLogger(PoloniexRateSource.class);
 
-    private BitfinexExchange btcRs;
-    private String preferedFiatCurrency;
```

Comment:
This was completely unused - Is this expected?

Reply:
Literally remnants of working with what i started with. IF not used then definitely see no point in keeping it.

Reply:
👍 will leave removed